StripeConnect/StripeConnectTests/Internal/AuthenticatedWebView/AuthenticatedWebViewManagerTests.swift

```diff
@@ -0,0 +1,163 @@
+//
+//  AuthenticatedWebViewManagerTests.swift
+//  StripeConnect
+//
+//  Created by Mel Ludowise on 10/15/24.
+//
+
+import AuthenticationServices
+@testable import StripeConnect
+import XCTest
+
+class AuthenticatedWebViewManagerTests: XCTestCase {
```

Comment:
I had to get creative to figure out how to unit test this, so lmk if you think this is easy enough to follow